static/app/views/discover/savedQuery/utils.tsx

```diff
@@ -242,3 +246,18 @@ export function displayModeToDisplayType(displayMode: DisplayModes): DisplayType
       return DisplayType.LINE;
   }
 }
+
+export function getDataset(
+  location: Location,
+  savedQuery: SavedQuery | undefined,
+  splitDecision?: string
```

Comment:
Would it be difficult to type `splitDecision` as a `SavedQueryDatasets`? I noticed it's being passed around as a string in the other files, and in this function we eventually force it to `SavedQueryDatasets`.